data_management/autopreprocess_testing/processing_pipeline.py

```diff
@@ -1,6 +1,17 @@
+# from unittest import case
+# from sklearn import preprocessing
```

Comment:
Please remove the commented out code.

Reply:
Can I do that with GitHub?  I have a revision on another branch.

Reply:
You technically can, but I think that would be bad practice. You could either finish your current commit on that branch and switch branches or stash the changes if the documents have already been added to the timeline.